.github/workflows/realease-images-from-main.yml

```diff
@@ -0,0 +1,42 @@
+name: Release named images
+# This action is meant to run on the main branch after a merge to release a named version of the image released
+# before with a hash as a tag.
+
+on:
+  push:
+    # ONLY main branch should be listed below!
+    branches: [ main ]
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '1 1 * * *'
+
+jobs:
+  retag:
+    name: Retag image & push
+    strategy:
+      matrix:
+        service_name: [ "sample-service", "practitioner-service", "trial-config-service", "role-service" ]
+    env:
+      GHCR: ghcr.io/ndph-arts
+      # We use 2 tags: latest (every PR merge to main) and nightly (main branch images every night)
+      TAG: ${{ github.event_name == 'schedule' && 'nightly' || 'latest' }}
```

Comment:
Should we be tagging "latest"  at this point or maybe "main" or "int", reason being latest is the default image often pulled - should this be the latest from main, or the latest release - that has gone through QA? Believe it's usually the latest "release".

Reply:
as agreed - main-latest / main-nightly